ultra/tests/evaluate_test.py

```diff
@@ -0,0 +1,124 @@
+# import unittest, ray, glob, gym, os
```

Comment:
Is this file intended to be commented out?

Reply:
it was throwing errors in the previous version so I commented out but we're gonna update these later.
so im gonna remove test folder for now and keep it just locally